packages/webpack/test/fixtures/main/node.js

```diff
@@ -0,0 +1,3 @@
+import dir from 'nodejs-package'
+
```

Comment:
Nit: Missing newline at EOF

Reply:
note: we do not automatically do this and we should.  iirc causes problems with the "dirty check", because of test fixture policies being regenerated.  we should either a) remove those from VCS or b) fix policy output such that it emits a newline at EOF.